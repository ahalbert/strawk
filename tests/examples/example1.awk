# hello
BEGIN { 
  x=1
  y=1 
}

/ +/ {
  x += length($0)
}


/#+/ {
  print "rect", x, x+length($0), y, y+1
  x+=length($0)
}

/\n/ {
	x=1
	y++ 
}
