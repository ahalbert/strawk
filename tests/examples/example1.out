rect 11 18 1 2
rect 10 19 2 3
rect 9 13 3 4
rect 15 20 3 4
rect 9 13 4 5
rect 14 18 4 5
rect 19 20 4 5
rect 8 12 5 6
rect 17 22 5 6
rect 7 11 6 7
rect 18 21 6 7
rect 7 15 7 8
rect 18 23 7 8
rect 7 11 8 9
rect 12 21 8 9
rect 7 11 9 10
rect 12 13 9 10
rect 15 16 9 10
rect 17 21 9 10
rect 7 9 10 11
rect 10 11 10 11
rect 13 16 10 11
rect 19 21 10 11
rect 7 10 11 12
rect 14 15 11 12
rect 17 20 11 12
rect 7 10 12 13
rect 14 16 12 13
rect 1 3 13 14
rect 4 5 13 14
rect 1 2 14 15
rect 3 7 14 15
rect 8 10 14 15
rect 1 3 15 16
rect 4 5 15 16
rect 6 8 15 16
